.github/workflows/ci.yaml

```diff
@@ -77,7 +77,7 @@ jobs:
             ${{ runner.os }}-build-${{ env.cache-name }}-
             ${{ runner.os }}-build-
             ${{ runner.os }}-
-      - run: cargo clippy --all -- -D warnings
+      - run: cargo clippy --all-targets --all-features -- -D warnings
```

Comment:
`--all` should have been kept (or replaced with its modern synonym `--workspace`). Now we aren’t linting the `ruff_dev` and `flake8-to-ruff` crates.

- #725